app/_components/nav-drawer/NavDrawer.tsx

```diff
@@ -0,0 +1,120 @@
+import {
```

Comment:
Can you add some basic component test for this new component, even a unit test that checks rendering is enough. We can add some integration tests to check navigation behavior as a follow up.

Reply:
Added basic render and close/open tests